troposphere/static/js/modals/instance/share.js

```diff
@@ -0,0 +1,18 @@
+import ModalHelpers from "components/modals/ModalHelpers";
+
+import InstanceShareModal from "components/modals/instance/InstanceShareModal";
+import actions from "actions";
+
+
+export default {
+    share: function(instance) {
+        ModalHelpers.renderModal(InstanceShareModal, { instance }, function({ user }) {
+            console.log(instance, user);
+            // Send action to subspace: share with instanceUsers (And remove with any who have been removed)
+            actions.InstanceActions.createShareRequest({
+                instance: instance,
+                user: user
+            });
```

Comment:
Todo: Need some help from a troposphere developer here -- newly created `InstanceAccessRequest` models are not added to the `InstanceAccessListView` in the `InstanceDetail` view.